README.md

```diff
@@ -32,3 +32,4 @@ Give a ⭐️ if you like this project!
 
 ### Acknowledgments
 * Microverse
+* Original design idea: [Cindy Shin](https://www.behance.net/adagio07)
```

Comment:
- [ ] Please include a live version of your project in your readme file. It is part of the requirements that you deploy the project online, so please work on this issue.

Reply:
Thank you